src/vivarium_gates_mncnh/data/loader.py

```diff
@@ -227,6 +228,36 @@ def load_lbwsg_exposure(
     return data
 
 
+def load_anc_proportion(
+    key: str, location: str, years: Optional[Union[int, str, list[int]]] = None
+) -> pd.DataFrame:
+    anc_proportion = load_standard_data(key, location, years)
+    year_start, year_end = 2021, 2022
+    lower_value = anc_proportion.loc[(year_start, year_end, "lower_value"), "value"]
+    mean_value = anc_proportion.loc[(year_start, year_end, "mean_value"), "value"]
+    upper_value = anc_proportion.loc[(year_start, year_end, "upper_value"), "value"]
+
+    try:
+        anc_proportion_dist = sampling.get_truncnorm_from_quantiles(
+            mean=mean_value, lower=lower_value, upper=upper_value
+        )
+        num_draws = 1000
```

Comment:
Should this be 500 or 1000? I know the new gbd rounds had 500 draws so maybe we want it to be 500? As long as we are consistent that is fine. I would move this constant to data_values.py and import it so we can stay consistent.